src/rulesets/Base/sort.jl

```diff
@@ -33,3 +37,54 @@ function rrule(::typeof(sort), xs::AbstractVector; kwargs...)
     end
     return ys, sort_pullback
 end
+
+#####
+##### `sortslices`
+#####
+
+function rrule(::typeof(sortslices), x::AbstractArray{<:Number}; dims::Integer, kw...)
```

Comment:
why not
```suggestion
function rrule(::typeof(sortslices), x::AbstractArray; dims::Integer, kw...)
```
?

Reply:
Oh right. The current version of `_zerolike_writeat` allows non-numbers, so this should be fine.

Reply:
My head hurts a bit thinking of `sortslices` on an array of arrays. But it does work, so it may as well be supported.